src/content/docs/en/reference/configuration-reference.mdx

```diff
@@ -180,13 +180,17 @@ In the example below, `astro dev` will start your server at `/docs`.
 }
 ```
 
-When using this option, all of your static asset imports and URLs should add the base as a prefix. You can access this value via `import.meta.env.BASE_URL`.
+When using this option, all of your static asset imports and URLs should add the base as a prefix. You can access this value via `import.meta.env.BASE_URL`. Default behavior for `import.meta.env.BASE_URL` includes the trailing slash.
```

Comment:
```suggestion
When using this option, all of your static asset imports and URLs should add the base as a prefix. You can access this value via `import.meta.env.BASE_URL`.
```
I think it's helpful to keep this sentence minimal, and then include all the mention of the slash below, rather than split it up into two pieces refering to the same idea!